Cargo.toml

```diff
@@ -3,10 +3,19 @@ name = "geotiff"
 description = "A GeoTIFF library for Rust"
 version = "0.0.2"
 edition = "2021"
-authors = ["Dominik Bucher <dobucher@ethz.ch>"]
+authors = [
+    "Dominik Bucher <dobucher@ethz.ch>",
+    "Gunnar Schulze <gunnar.schulze@gmail.com>"
+]
 repository = "https://github.com/georust/geotiff"
 
 [dependencies]
+delaunator = "1.0"
+geo-types = { version = "0.7", features = ["approx", "rstar_0_12"] }
 num_enum = "0.7"
 num-traits = "0.2"
+rstar = "0.12"
```

Comment:
Since in this PR it looks like you're only using an rtree statically (e.g. you don't need to insert items after the initial bulk load), you may want to consider a static index like https://github.com/kylebarron/geo-index, which I've benchmarked to be [reasonably faster](https://github.com/kylebarron/geo-index#benchmarks) and much more memory efficient than rtree.